ophyd/__init__.py

```diff
@@ -7,3 +7,5 @@
 
 from .controls import *
 from .session import get_session_manager
+
+__version__ = 'v0.0.3.post0'
```

Comment:
Because we don't have versioneer set up yet and this will make sure it sorts after `0.0.3`